toolkits/global/packages/global-ad/scss/50-components/ad.scss

```diff
@@ -0,0 +1,70 @@
+// pubgrade (ads) put a 1px image as first child of the body which we need to hide
+body > img[src*=pbgrd] {
+	display: none;
+}
+
+.c-ad {
+	@include font-size($ad--font-size);
+	font-family: $ad--font-family;
+	display: none;
+
+	.js & {
+		@include media-query('md') {
+			display: block;
+		}
+	}
+}
+
+.c-ad iframe {
+	border: none;
+	overflow: auto;
+	vertical-align: top;
+}
+
+.c-ad--leaderboard {
+	background-color: $ad--leaderboard-bg-color;
+	padding: $ad--leaderboard-padding;
+	text-align: center;
+}
+
+.c-ad--leaderboard iframe {
+	max-width: 970px;
+	height: 90px;
+}
+
+.c-ad--leaderboard .c-ad__inner {
+	display: inline-block;
+	vertical-align: top;
+}
+
+.c-ad--skyscraper iframe {
+	width: 160px;
+	height: 600px;
+}
+
+.c-ad--mpu {
+	background-color: $ad--mpu-background-color;
+	margin-top: $ad--mpu-margin-top;
+	padding-top: $ad--mpu-padding-top;
+	padding-bottom: $ad--mpu-padding-bottom;
+}
+
+.c-ad--mpu .c-ad__label {
+	margin-bottom: $ad--mpu-label-margin-bottom;
```

Comment:
Can this be more generic?
Any ad that has a label, the label should have the same spacing underneath, no?

Reply:
got rid of this and label spacing is consistent